specification/deviceupdate/resource-manager/Microsoft.DeviceUpdate/preview/2020-03-01-preview/deviceupdate.json

```diff
@@ -1613,13 +1613,15 @@
           "description": "Remote private endpoint details.",
           "$ref": "#/definitions/RemotePrivateEndpoint"
         },
-        "provisioningState": {
-          "$ref": "#/definitions/PrivateEndpointConnectionProxyProvisioningState",
-          "description": "The provisioning state of the private endpoint connection proxy resource."
-        },
         "status": {
           "description": "Operation status.",
           "type": "string"
+        },
+        "properties": {
```

Comment:
Semantic error is caused by extra properties layer is added and need to be removed.

If eTag, remotePrivateEndpoint, status properties doesn't have to be in top level, then you can change model `PrivateEndpointConnectionProxy` removing line#1598. replace it reference it in properties bag.

Reply:
Reverted the last commit.

Reply:
eTag and remotePrivateEndpoint unfortunately have to be top level

Reply:
Just move this whole of part 1620-1624 to line#1601 should work.